samples/sensor/ccs811/boards/efr32mg_sltb004a.overlay

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2020 Christian Taedcke
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c1 {
+	location-sda = <GECKO_LOCATION(6) GECKO_PORT_B GECKO_PIN(6)>;
+	location-scl = <GECKO_LOCATION(6) GECKO_PORT_B GECKO_PIN(7)>;
+	status = "okay";
```

Comment:
Is there any reason not to put everything except the "okay" line in the board-level device tree? That would make it easier for other applications to enable the sensor.

Reply:
Yes, the reason is described in the pr description. The SoC has only 2 I2C controller, but on-board sensors are connected to 4 different pin locations, so it is not possible to support every on-board sensor/I2C bus at the same time. Depending on the application only a certain sub-set of the sensors can be used.
In the current device tree the two I2C controller are enabled. But one controller is configured to be connected to the external board connector. The other i2c controller is connected to three other sensors (BMP280, Si7021 and Si1133).

Reply:
Ahh, yes, i will move this into the board file, except the "okay". I will ping when i am done (also rebase to current master)

Reply:
I added the sensor to the board dts file, but left the i2c bus disabled. No other device is connected to that i2c pins. The devices on the i2x pins, that i removed (PC4 and PC5) are not supported by zephyr.